### PR TITLE
Add Docked/Handheld text on status bar

### PR DESCRIPTION
**Pull request by HorrorTroll**
Since we have dynamic Docked/Handheld already, so i try to add Docked/Handheld text on status bar. 

The preview is on this gif (it was an old gif, but the function is the same)
ezgif com-optimize

### CODE REVIEW

File: Ryujinx/Ui/GLRenderer.cs [hunk 1]
```diff
@@ -326,6 +326,11 @@ public void Render()
                     _device.ProcessFrame();
                 }
 
+                string dockedMode = "Unknown";
+
+                if (ConfigurationState.Instance.System.EnableDockedMode == true)  dockedMode = "Docked";
+                if (ConfigurationState.Instance.System.EnableDockedMode == false) dockedMode = "Handheld";
```
Comment (AcK77) [inline Ryujinx/Ui/GLRenderer.cs:332]: This should be simplified by a ternary condition:

```csharp
string dockedMode = ConfigurationState.Instance.System.EnableDockedMode ? "Docked" : "Handled";
```

**Comment by jduncanator** [comment]
I think this should go next to Vsync, which is our other "dynamically toggle-able setting".

**Comment by HorrorTroll** [comment]
Already simplified & move next to V-Sync like suggest above
image

File: Ryujinx/Ui/StatusUpdatedEventArgs.cs [hunk 2]
```diff
@@ -5,16 +5,19 @@ namespace Ryujinx.Ui
...
 
-        public StatusUpdatedEventArgs(bool vSyncEnabled, string hostStatus, string gameStatus, string gpuName)
+        public StatusUpdatedEventArgs(bool vSyncEnabled, string dockedMode, string hostStatus, string gameStatus, string gpuName)
         {
             VSyncEnabled = vSyncEnabled;
+            DockedMode   = dockedMode;
             HostStatus   = hostStatus;
             GameStatus   = gameStatus;
             GpuName      = gpuName;
+
```
Comment (Thog) [inline Ryujinx/Ui/StatusUpdatedEventArgs.cs:20]: nit useless space here

Comment (HorrorTroll) [reply Ryujinx/Ui/StatusUpdatedEventArgs.cs]: Done 👍

**Pull request merged**